### DISCUSSION TITLE
Data Request: add global Administrative Boundaries dataset to STAC catalog

### DISCUSSION THREAD

**Discussion by julietcohen** [microsoft/PlanetaryComputer#25] [Data request]
Hello, I’m a master’s student working with the MOSAIKS machine learning algorithm on Planetary Computer for my capstone project. I would like to request that Planetary Computer add a global Administrative Boundary dataset to the STAC catalog. I am hoping for a dataset similar to the [FAO GAUL dataset on Google Earth Engine](https://developers.google.com/earth-engine/datasets/catalog/FAO_GAUL_2015_level2#description). I would really appreciate having this dataset in Planetary Computer as it would help streamline my project that applies MOSAIKS to study crop yields in several different countries. The STAC API has worked well for my team using Landsat data so far, and we would like to use it for all our spatial data if possible. If the Administrative Boundary data can be added to the STAC catalog, please let me know a general timeline for when you think it will be available. Thank you very much!

**Comment by TomAugspurger** [comment]
Thanks. I'll get this added to the backlog.

Out of curiosity, do you have any experience with the UN's SALB (https://www.unsalb.org/)? That's another administrative boundaries dataset I've been meaning to add.

**Reply by julietcohen** [reply]
Thank you! No, I was not aware of that dataset, but it looks like a great fit for my project. I'll be sure to share it with my team.
